### ISSUE TITLE
Could this ship with Typescript types

### ISSUE THREAD

**Issue by codingedgar** [fluture-js/fluture-sanctuary-types#51]
Hi ! Thanks a lot for this integration with Sanctuary, makes everything better 👍 

I'm using typescript and have been working pretty well for Fluture and Sanctuary so far, however there are no `@types/fluture-sanctuary-types` library, with makes ts throw a `Could not find a declaration file for module`

i made a workaround by declaring the tipes in a file:

```ts
// ./src/misssingDeclarations/fluture-sanctuary-types.d.ts
declare module 'fluture-sanctuary-types' {
    export const env: ReadonlyArray<any>;
}
```

and 

```ts
// tsconfig.json
{
  "compilerOptions": {
    "baseUrl": "./",
    "paths": {
      "fluture-sanctuary-types": [
        "./src/misssingDeclarations/fluture-sanctuary-types.d.ts"
      ]
    }, 
}
```
But seems there are no official typescript package.
Sorry for bothering let me know if there is any way i could contribute. 🙏

**Comment by Avaq** [comment]
> But seems there are no official typescript package.
> Sorry for bothering let me know if there is any way i could contribute.

If you like you could open a PR, adding an `index.d.ts` file to this package. I would not be opposed to adding types to the module.

For now I guess the members of `env` would indeed best be types as `any`. Later we could upgrade them to use the typings that are being added to Sanctuary.

**Comment by codingedgar** [comment]
I like the idea, would do 👍

**Comment by codingedgar** [comment]
@Avaq I made the pull request, please let me know if there's anything i could improve 👍

**Comment by Avaq** [comment]
Types are included in version 4.1.1 :)
